src/main/java/uk/gov/justice/digital/hmpps/keyworker/batch/CamelQuartzConfigurationFactory.kt

```diff
@@ -0,0 +1,104 @@
+package uk.gov.justice.digital.hmpps.keyworker.batch
+
+import org.apache.camel.CamelContext
+import org.apache.camel.ProducerTemplate
+import org.apache.camel.component.quartz2.QuartzComponent
+import org.apache.camel.spring.boot.CamelContextConfiguration
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.util.*
+
+/* requires postgres db - does not work with in memory database */
+@Configuration
+@ConditionalOnProperty(name = ["quartz.enabled"])
+internal class CamelQuartzConfigurationFactory(
+        @Autowired
```

Comment:
kotlin formatting seems off here.  Should be using `ktlint` and getting it to format automatically.